src/assertions/utils/globalTypeChecker.ts

```diff
@@ -0,0 +1,4 @@
+type GlobalWithExpectKey = { expect: any };
+export const isGlobalWithExpectKey = (
+  global: any
+): global is GlobalWithExpectKey => "expect" in global;
```

Comment:
You can let `any` for this one because it's little complicated and unecessary to type expect.

But for the next time you want to write a typeguard which test a attribute existence I recommand you this one
```ts
export const hasAttributes = <T extends string>(
    element: unknown,
    attributes: T[]
): element is Record<T, unknown> => {
    if (element === undefined || element === null) {
        return false;
    }
    return attributes.every((attribute) =>
        Object.prototype.hasOwnProperty.call(element, attribute)
    );
};
```
from [the fresh article of Fatine about typeguards](https://blog.theodo.com/2022/01/typescript-replace-as-typeguards/)